models/like.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  var like = sequelize.define('like', {
+    user_id: DataTypes.INTEGER,
+    model_name: DataTypes.STRING,
+    model_id: DataTypes.STRING
+  }, 
+  {
+    underscored: true
+  },
+  {
+    classMethods: {
+      associate: function(models) {
+        // associations can be defined here
+      },
+      get_like_by_user: function(post_id, user_id) {
```

Comment:
did we decide to restrict this to posts? if so, we should also set the model_name to "post". If not, we should add the model name as an arg so we can use this also for liking statues, comments(?)